website/docs/cloud-docs/api-docs/policy-set-params.mdx

```diff
@@ -36,7 +36,7 @@ page_title: Policy Set Parameters - API Docs - Terraform Cloud
 
 # Policy Set Parameters API
 
--> **Note:** Sentinel policies are a paid feature, available as part of the **Team & Governance** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing).
+-> **Note:** Sentinel and OPA policies are paid features, available as part of the **Team & Governance** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing).
```

Comment:
Also would it be possible to add a 1-3 sentence description of what Policy Set parameters mean/are? I'm having trouble understanding exactly what this is and how it relates to all the other APIs. For example, don't we already have one for policy sets? Maybe we should link to that API and explain what's different about this one.

```suggestion
-> **Note:** Sentinel and OPA policies available in the [Terraform Cloud **Team & Governance** tier](https://www.hashicorp.com/products/terraform/pricing), and OPA policies are in beta.
```